api/exec/add_env.go

```
@@ -29,3 +30,27 @@ func AddEnv(workspaceId string, applicationId string, key string, value string)
+func AddEnvWithLabels(workspaceId string, labels []string, key string, value string) error {
+	header := make(map[string]string)
+	accessToken, err := GetAccessToken()
+	if err != nil {
+		return err
+	}
+	header["Authorization"] = "Bearer " + accessToken
+
+	body := map[string]string{}
+	body[key] = value
+	envReq := envRequest{EnvList: body}
+	requestJson, err := json.Marshal(envReq)
```

golangci-lint

ineffectual assignment to err

(ineffassign)